### ISSUE TITLE
Array of strings to automatically create highlights

### ISSUE THREAD

**Issue by SimeriaIonut** [agentcooper/react-pdf-highlighter#245]
Is there an easy way to have an array of strings and find them in the PDF and create a highlight for that found string? I don't see a way to look for strings in the PDF and also to get back the boundingRect for it.

Thanks!

**Comment by straeter** [comment]
@SimeriaIonut this was already discussed here: https://github.com/agentcooper/react-pdf-highlighter/issues/176

**Comment by SimeriaIonut** [comment]
@straeter that doesn't seem to return any positioning of the found string (I would need that to create the highlight programatically). Or am I missing something?

**Comment by straeter** [comment]
@SimeriaIonut yes you are right, it just highlights the text without giving back the positions. Actually, I am also working on a solution to get the positions from search queries, it seems to be quite tricky. Let's keep each other updated on this

**Comment by AlfredMadere** [comment]
Any progress on this?
